test/generators/randoms_test.cpp

```diff
@@ -17,6 +18,26 @@ TEST(GeneratorsRandoms, RandomsGenerator) {
   }
 }
 
+TEST(GeneratorsRandoms, GaussianRandomsGenerator) {
+  lin::internal::RandomsGenerator rand;
+  double mean = 0.0L;
+  double std = 0.0L;
+  uint N = 10000000;
+  for (lin::size_t i = 0; i < N; i++) {
+    double x = rand.gaussian();
+    mean += x;
+    std += x*x;
+  }
+  std = sqrt(std/N);
+  mean = mean/N;
+
+  ASSERT_LE(mean,0.01);
+  ASSERT_GT(mean,-0.01);
+
+  ASSERT_LE(std,1.01);
+  ASSERT_GT(std,0.99);
+}
+
```

Comment:
Nice! Matters less in unit tests, but using `std::sqrt` would be better here too!